plugins/hydra_joblib_launcher/README.md

```diff
@@ -0,0 +1,45 @@
+# Hydra Joblib Launcher
+
+This plugin provides a launcher for parallel tasks based on [`Joblib.Parallel`](https://joblib.readthedocs.io/en/latest/parallel.html).
+
+The configuration for this launcher is packaged with the plugin:
+`hydra_plugins/hydra_joblib_launcher/conf/hydra/launcher/joblib.yaml`
+```yaml
+hydra:
+  launcher:
+    class: hydra_plugins.hydra_joblib_launcher.HydraJoblibLauncher
+    params:
+      joblib: ${hydra.joblib}
+
+  joblib:
+    n_jobs: -1  # maximum number of concurrently running jobs. if -1, all CPUs are used
+    backend: null  # allows to hard-code backend, otherwise inferred based on prefer and require
+    prefer: processes  # processes or threads, soft hint to choose backend
+    require: null  # null or sharedmem, sharedmem will select thread-based backend
+    verbose: 0  # if greater than zero, prints progress messages
+    timeout: null  # timeout limit for each task
+    pre_dispatch: 2*n_jobs  #  number of batches to be pre-dispatched
+    batch_size: auto  # number of atomic tasks to dispatch at once to each worker
+    temp_folder: null  # folder used for memmapping large arrays for sharing memory with workers
+    max_nbytes: 1M  # thresholds size of arrays that triggers automated memmapping
+    mmap_mode: r  # memmapping mode for numpy arrays passed to workers
+```
+
+All arguments specified in `joblib` are passed to `Joblib.Parallel` (see [`Joblib.Parallel` documentation](https://joblib.readthedocs.io/en/latest/parallel.html) for details). `n_jobs` defaults to -1, which means that all available CPUs may be used. `prefer` defaults to `processes`, depending on the application, `threads` can be an alternative. 
+
+An example application using the plugin is provided in `plugins/hydra_joblib_launcher/example`. It overwrites the launcher used by Hydra.
```

Comment:
Can you make the example app a link to the app dir?

```suggestion
An example application using this launcher is provided in `plugins/hydra_joblib_launcher/example`.  It changes the default launcher Hydra is using to this one.
```

Reply:
Done

Reply:
You didn't delete the line this was editing.

Reply:
Changed `README.md` to say:

```markdown
An [example application](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_joblib_launcher/example) using this launcher is provided in `plugins/hydra_joblib_launcher/example`. It changes the default launcher Hydra is using to this one.
```

Maybe I didn't get it, what is your suggestion?